### ISSUE TITLE
500 error when trying to setup reverse-proxy with SSL (Apache)

### ISSUE THREAD

**Issue by FeehGamer** [louislam/uptime-kuma#632]
**Is it a duplicate question?**
Please search in Issues without filters: https://github.com/louislam/uptime-kuma/issues?q=
I couldn't find any issue with this problem

**Describe your problem**
Hello, I'm trying to setup Reverse-Proxy with SSL using Apache for Uptime-Kuma
But I get 500 Internal Server Error, I double-checked that I did everything correctly but the error is still there

**Info**
Uptime Kuma Version: 1.7.3
Using Docker?: No
Docker Version: //
Node.js Version (Without Docker only): v16.11.0
OS: Ubuntu 2004
Browser: Chrome

**Comment by FeehGamer** [comment]
Hello, I have another problem, now I can't use dashboard because it says this:
Cannot connect to the socket server. [Error: websocket error] Reconnecting...

**Comment by louislam** [comment]
If localhost is working, it is usually because of missing the upgrade headers.

**Comment by FeehGamer** [comment]
Yes, localhost is working, could you please tell me how to add the upgrade headers?
